# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/tests/file_cross_frame.html
# Commit: f45996fe15b2
# Full Hash: f45996fe15b23696c88f85a6b154146d3717fc58
# Author: Andrew McCreight <continuation@gmail.com>
# Date: 2021-05-06 03:49:11
# Regressor Bug: 1618547
# File Overlap Count: 1
# Description:
#   Bug 1618547 - Support Fission in the process priority manager. r=kmag,gsvelto
#   
#   With Fission, there can be multiple BrowserParents in a single tab, so
#   this patch moves the tracking of active tabs onto the top BrowsingContext
#   in a tab. If the priority of a top BC is changed, then the activity
# ==============================================================================

diff -r 0e0f90271853 -r f45996fe15b2 dom/ipc/tests/file_cross_frame.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/dom/ipc/tests/file_cross_frame.html	Wed May 05 20:29:14 2021 +0000
@@ -0,0 +1,12 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Different-origin iframe</title>
+</head>
+<body>
+<iframe id="testIFrame" src="http://example.org/browser/dom/ipc/tests/file_dummy.html"></iframe>
+<i>I am a web page</i>
+</div>
+</body>
+</html>